spec/lib/spree/active_shipping/bogus_carrier.rb

```diff
@@ -3,12 +3,15 @@ module ActiveShipping
     # Bogus carrier useful for testing.  For some reasons the plugin version of this class does not work
     # properly (it fails to return a RateResponse)
     class BogusCarrier < ::ActiveShipping::Carrier
+      include RSpec::Mocks::ExampleMethods
```

Comment:
We shouldn't need to do this. It seems like we have two versions of rspec-mocks, and one is locked because of this https://github.com/solidusio-contrib/solidus_active_shipping/blob/master/solidus_active_shipping.gemspec#L25
Can you try removing that from the gemspec since we should not need it, removing the `Gemfile.lock` and bundling.